packages/components/tooltip/README.mdx

```diff
@@ -0,0 +1,54 @@
+# @contentful/f36-tooltip
+
+---
+
+title: 'Tooltip'
+type: 'component'
+status: 'stable'
+slug: /components/tooltip/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/tooltip'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-tooltip--default'
+
+---
+
+Tooltips are very useful to communicate extra information related to an element on the screen. The information should be contextual, useful, and nonessential.
+
+- [How to use Tooltip](#how-to-use-Tooltip)
+- [Component variations](#component-variations)
+- [Code examples](#code-examples)
+- [Content recommendations](#content-recommendations)
+- [Accessibility](#accessibility)
+
+## How to use Tooltip
+
+- Even though it's possible to pass any ReactNode as a content of the Tooltip,
+  we do not recommend passing very complex components try to use only [Phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content)
+
+## Code examples
+
+```jsx
+import { Tooltip } from '@contentful/f36-components';
+<Tooltip
+  place="top"
+  id="tip1"
+  targetWrapperClassName="targetWrapperClassName"
+  content="Hi I am a Tooltip"
+>
+  <TextLink>Hover me</TextLink>
+</Tooltip>;
+```
+
+## Content recommendations
+
+- Use short and clear messages as the Tooltip’s content
+
+## Accessibility
+
+- Add `aria-describedby` property to the target element to make the contents of the tooltip more accessible
```

Comment:
Can we make it automatic? 

https://chakra-ui.com/docs/overlay/tooltip - generates id automatically, so consumers don't need to bother about adding `aria-describedby` manually.

Reply:
sure, good example